corehq/apps/dump_reload/management/commands/dump_domain_data.py

```diff
@@ -55,7 +55,13 @@ def handle(self, domain_name, **options):
             filename = _get_dump_stream_filename(dumper.slug, domain_name, self.utcnow)
             stream = self.stdout if console else gzip.open(filename, 'wt')
             try:
-                meta[dumper.slug] = dumper(domain_name, excludes, includes).dump(stream)
+                result = dumper(domain_name, excludes, includes).dump(stream)
+                if dumper == SqlDataDumper:
+                    counts = result[0]
+                    meta[dumper.max_pk_slug] = result[1]
+                else:
+                    counts = result
```

Comment:
It would be good to keep the return value consistent between the different types of 'dumpers'. For couch dumpers the value could be None. 

I think a simple dataclass would make sense for this:

```
@dataclass
class DumpResult:
    count: int
    max_pk: int|None = None
```